src/order/category/FinBoolAlg.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.BoolAlg
+import order.category.FinDistribLattice
+
+/-!
+# The category of finite boolean algebras
+
+This defines `FinBoolAlg`, the category of finite boolean algebras.
+-/
+
+universes u v
+
+open category_theory
+
+/-- The category of finite boolean algebras with bounded lattice morphisms. -/
+structure FinBoolAlg :=
+(to_BoolAlg : BoolAlg)
+[is_fintype : fintype to_BoolAlg]
+
+namespace FinBoolAlg
+instance : has_coe_to_sort FinBoolAlg Type* := ⟨λ X, X.to_BoolAlg⟩
+instance (X : FinBoolAlg) : boolean_algebra X := X.to_BoolAlg.str
+
+attribute [instance]  FinBoolAlg.is_fintype
+
+/-- Construct a bundled `FinBoolAlg` from a `bounded_order` `distrib_lattice`. -/
+def of (α : Type*) [boolean_algebra α] [fintype α] : FinBoolAlg := ⟨⟨α⟩⟩
+
+instance : inhabited FinBoolAlg := ⟨of bool⟩
+
+instance : large_category.{u} FinBoolAlg :=
```

Comment:
Can you inherit the structure from `BoolAlg` here? And that forgetful functor is also missing.

Reply:
I'm not sure what you're referencing here!

Reply:
Ah I thought I was replying to https://github.com/leanprover-community/mathlib/pull/11677/files/2861ff07e6ce6e9f7b7b4f1527679565ee2de3b0#diff-7b13464e7911e9d0454c0351ce05ea4037f70919e81adb7b2d02064495ee3668 !

Reply:
It seems we're missing `induced_category.large_category`. Is it not true?

Reply:
So I managed to use `induced_category` when I already had explicit maps like `FinPartialOrder.to_PartialOrder`. Is it worth extending to places where I don't typically from `LinearOrder` to `PartialOrder`?